libweasyl/libweasyl/staff.py

```diff
@@ -54,17 +52,16 @@
 """ Purely cosmetic group for users who contribute to site development. """
 
 
-def _init_staff(staff_config_path):
+def _init_staff(staff_config_dict):
     """
     Loads staff from a yaml config file.
 
     Parameters:
         staff_config_path: String path to a yaml file specifying staff.
     """
-    with open(staff_config_path) as config_file:
-        staff = yaml.safe_load(config_file)
-    DIRECTORS.update(staff['directors'])
-    TECHNICAL.update(DIRECTORS, staff['technical_staff'])
-    ADMINS.update(DIRECTORS, TECHNICAL, staff['admins'])
-    MODS.update(ADMINS, staff['mods'])
-    DEVELOPERS.update(staff['developers'])
+
+    DIRECTORS.update(staff_config_dict['directors'])
```

Comment:
I’d prefer this overwrote each set, i.e.

```python
DIRECTORS = frozenset(directors)
TECHNICAL = DIRECTORS | frozenset(technical)
ADMINS = DIRECTORS | TECHNICAL | frozenset(admins)
…
```

but maybe there’s a good reason for not doing that. Can someone else weigh in?